website/docs/governance/team.md

```diff
@@ -4,28 +4,28 @@ title: The SemApps team
 
 - **Core developpers**
     - [Sébastien Rosset](https://www.linkedin.com/in/sebastien-rosset-reconnexion/) : Software developer 
-        - Expertises: Semantic web, ActivityPub, JavaScript, React.js, Moleculer.js. 
+        - _Expertises_ : Semantic web, ActivityPub, JavaScript, React.js, Moleculer.js. 
     - [Simon Louvet](https://www.linkedin.com/in/simon-louvet-a9842018/) : Software developer 
-        - Expertises : Semantic web, SOLID spécification, Moleculer.js. 
+        - _Expertises_ : Semantic web, SOLID spécification, Moleculer.js. 
     - [Niko Bonnieure](https://github.com/nikoPLP) : Software developer 
-        - Expertises: Java/J2EE, web-ACL, Moleculer.js, React.js
+        - _Expertises_ : Java/J2EE, web-ACL, Moleculer.js, React.js
         
 - **Scrum master**
     - [Pierre Bouvier](https://www.linkedin.com/in/bouviermullerp/) : Agile project manager
-        - Expertises : Agility, lean management, semantic web & Linked Open Data
+        - _Expertises_ : Agility, lean management, software & community management.
 
 - **Product Owners** 
     - [Guillaume Rouyer](https://www.linkedin.com/in/guillaume-rouyer-paris/) : Project designer
-        - Expertises : Prospective, political philosophy, ecology & econnomics, complex systems, community management, 
+        - _Expertises_ : Prospective, political philosophy, ecology & econnomics, complex systems, community management, 
     - [Gabriel Henry](https://www.linkedin.com/in/gabriel-henry-87915690/?originalSubdomain=fr) :
-        - Expertises : Coaching, Innovation, project & community management, IT, social innovation, social economy, IT 
+        - _Expertises_ : project management, quality management, IT, business analysis, coaching, social innovation, social economy
     
 - **Technical advisors** 
     - [Thomas Francart](http://www.sparna.fr/qui-suis-je/) : Software architect & semantic web expert
-        - Expertises: RDF, OWL, triplestores, Sesame, SKOS,inferences, linked data, open data, inetroperablity. 
+        - _Expertises_ : RDF, OWL, triplestores, Sesame, SKOS,inferences, linked data, open data, inetroperablity. 
```

Comment:
- [Thomas Francart](http://www.sparna.fr/qui-suis-je/) : Knowledge Graph and Semantic Web senior consultant
        - _Expertises_ : RDF, OWL, SPARQL, SHACL / SKOS, FRBR, CIDOC-CRM, RiC-O / linked data, interoperability / Jena, RDF4J, GraphDB, Virtuoso / [SKOS Play](http://labs.sparna.fr/skos-play/), [SHACL Play](http://shacl-play.sparna.fr/) and [blog](http://blog.sparna.fr/).